### PR TITLE
clarity on x-extensible-enum usage for events 

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

File: chapters/compatibility.adoc [hunk 1]
```diff
@@ -55,9 +55,10 @@ services in a backward-compatible way:
 * Enum ranges cannot be extended when used for output parameters — clients may
   not be prepared to handle it. However, enum ranges can be extended when used
   for input parameters.
-* Use {x-extensible-enum}, if range is used for output parameters and likely to
+* Use {x-extensible-enum} (see <<112, rule 112>>), if range is used for output parameters and likely to
   be extended with growing functionality. It defines an open list of explicit
-  values and clients must be agnostic to new values.
+  values and clients must be agnostic or provide default behavior for unknown 
+  values (see <<108, rule 118>>).
```
Comment (a1exsh) [inline chapters/compatibility.adoc:61]:
```suggestion
  values (see <<108, rule 108>>).
```
I guess a typo?  Rule 118 is about `snake_case`: https://opensource.zalando.com/restful-api-guidelines/#118

**Comment by a1exsh** [comment]
When I think about it, the `x-extensible-enum` gives a stronger signal to the event publishers: namely, the set of currently accepted values.

For the event consumers, in turn, it only signifies the list of currently most commonly found values, but does not restrict the future or the past set of possible values in any way.  In that regard it's not much different from a free-form string with `examples` provided.

**Comment by a1exsh** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Comment by ePaul** [comment]
@tfrauenstein wanted to extend the PR as discussed in the Meeting ([internal link](https://docs.google.com/document/d/1NahSuMC0LRwFTTrln2m0iDLq0nHzGmF59Gv7xBhKPfQ/edit#bookmark=id.h23t4po7q2os))

**Comment by tfrauenstein** [comment]
done

File: chapters/compatibility.adoc [hunk 2]
```diff
@@ -315,14 +314,21 @@ delivery_methods:
     - EMAIL
 ----
 
-*Note:* {x-extensible-enum} is not JSON Schema conform but will be ignored by
-most tools.
+*Note:* {x-extensible-enum} is a proprietary extension of the JSON Schema standard that 
+will be ignored by most tools.
```
Comment (tkrop) [inline chapters/compatibility.adoc:318]: Does it make sense to mention that the swagger UI tooling is showing it in the latest versions?

File: chapters/compatibility.adoc [hunk 3]
```diff
@@ -315,14 +314,21 @@ delivery_methods:
...
+full type-range specification that is validated by the service.
+
 *Important:* Clients must be prepared for extensions of enums returned with server responses, i.e. 
 must implement a fallback / default behavior to handle unknown new enum values -- see <<108>>. 
 API owners must take care to extend enums in a compatible way that does not change the 
 semantics of already existing enum values, for instance, do not split an old enum value 
-into two new enum values. 
+into two new enum values. Services should only extend {x-extensible-enum} ranges, and only accept 
+and return values listed in the latest API version which needs to be updated first before 
+providing new values -- see also <<107>>.
```
[the remark this answers is not in the document]
Comment (tfrauenstein) [reply chapters/compatibility.adoc:333]: done -- pls. check.

File: chapters/compatibility.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/compatibility.adoc:318]: done

File: chapters/compatibility.adoc [hunk 4]
```diff
@@ -315,14 +314,21 @@ delivery_methods:
     - EMAIL
 ----
 
-*Note:* {x-extensible-enum} is not JSON Schema conform but will be ignored by
-most tools.
+*Note:* {x-extensible-enum} is a proprietary extension of the JSON Schema standard that 
+is e.g. visible via Swagger UI and ignored by most tools.
```
Comment (tkrop) [inline chapters/compatibility.adoc:318]:
```suggestion
is e.g. visible via Swagger UI, but ignored by most other tools.
```

File: chapters/compatibility.adoc [hunk 5]
```diff
@@ -315,14 +314,21 @@ delivery_methods:
...
 
 See <<240>> about enum value naming conventions.
 
+Note, {x-extensible-enum} is a different concept than JSON schema `examples` which is 
+just a list of few example values, whereas {x-extensible-enum} defines all valid
```
Comment (tkrop) [inline chapters/compatibility.adoc:323]:
```suggestion
just a list of a few example values, whereas {x-extensible-enum} defines all valid 
```

**Comment by tkrop** [comment]
👍

**Pull request merged**